modules/manage/partials/mountable-topics.adoc

```diff
@@ -7,7 +7,14 @@ For topics with Tiered Storage enabled, you can unmount a topic to safely detach
 include::shared:partial$enterprise-license.adoc[]
 ====
 
+ifndef::env-cloud[]
 . xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
+endif::[]
+ifdef::env-cloud[]
+. xref:manage:rpk/rpk-install.adoc[Install `rpk`] to run cluster mount and unmount operations on the command line, or xref:manage:api/cloud-api-authentication.adoc[authenticate] to the Cloud API and execute these operations programmatically.
++
+If using the Cloud API, you'll make requests against Data Plane APIs. Make sure you xref:manage:api/cloud-dataplane-api.adoc#get-data-plane-api-url[retrieve the Data Plane API URL] and use this URL to perform a mount or unmount operation.
+endif::[]
 . Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
```

Comment:
@mattschumpert how should Cloud users make sure that they meet this prereq? Can they toggle Tiered Storage on a per-topic basis and is that what we should say for the Cloud version of this page?